core/frontend/src/RealityDataSourceTilesetUrlImpl.ts

```diff
@@ -107,7 +112,7 @@ export class RealityDataSourceTilesetUrlImpl implements RealityDataSource {
 
   /** Returns the tile URL. If the tile path is a full URL, it is returned as is. Otherwise, the base URL is prepended to the tile path. */
   private getTileUrl(tilePath: string){
-    return this.isValidURL(tilePath) ? tilePath : this._baseUrl + tilePath;
+    return this.isValidURL(tilePath) ? tilePath : this._baseUrl + tilePath + this._searchParams;
```

Comment:
Shouldn't it be
```suggestion
    return (this.isValidURL(tilePath) ? tilePath : this._baseUrl + tilePath) + this._searchParams;
```
?

Reply:
I'm not sure if this is correct. The search parameters only get lost above in `setBaseUrl()` when the original url is split to obtain a base url that would be prepended to a relative tile path. I believe this means the search parameters would then only need to be re-inserted in the case of a relative tile path, so the second option in the conditional. The tile path being a full url should mean that it has everything it needs for a successful request, and therefore doesn't need the search parameters appended to them.

Reply:
> The tile path being a full url **_should_** mean that it has everything it needs for a successful request, and therefore doesn't need the search parameters appended to them.

(emphasis above is mine)

Has this theory been tested?

Reply:
This has been done in the latest commits

Reply:
Looks good, thanks Andre. @danieliborra When you have a chance, could you look and possibly resolve this comment?